meta-agl-tune/README.md

```diff
@@ -0,0 +1,37 @@
+This layer contains files that were:
+ - copied as-is or adopted from AGL layer meta-agl/meta-agl-bsp/meta-rcar-gen3;
+ - developed by EPAM.
+
+We can't use meta-agl-bsp due to lots of conflicts in handling of prebuilt
+binaries and duplication of functionality with out layers.
```

Comment:
"our layers", maybe?

Reply:
Fixed in the text and in the commit message.